source/_integrations/nanoleaf.markdown

```diff
@@ -50,3 +51,58 @@ name:
 4. The output should include the auth token like *{"auth_token":"xxxxxxxxxxxxxxxxxxxxx"}*, copy the resulting token into your configuration
 
 If you get a 403 Forbidden message, you probably did not press the *ON* button long enough. The time-frame to get a valid token is only 30 seconds, so you have to be quick to issue the curl request.
+
+### Getting useful values for automations
+
+First set up some effects with the Nanoleaf app, and have them installed on the device.
+
+Once installed, go to "Developer Tools" and find the state of the device, eg `light.bedroom_triangles`. You should end up with something like this:
```

Comment:
Why does one need the developer tools? The effects are available on the light entity itself, right?

```suggestion
Once installed, go to "Developer Tools" and find the state of the device, e.g., `light.bedroom_triangles`. You should end up with something like this:
```